applications/StructuralMechanicsApplication/custom_elements/membrane_element.cpp

```diff
@@ -1315,6 +1315,36 @@ void MembraneElement::CalculateDampingMatrix(
         GetGeometry().WorkingSpaceDimension()*GetGeometry().size());
 }
 
+const Parameters MembraneElement::GetSpecifications() const
+{
+    const Parameters specifications = Parameters(R"({
+        "time_integration"           : ["static","implicit","explicit"],
+        "framework"                  : "lagrangian",
+        "symmetric_lhs"              : true,
+        "positive_definite_lhs"      : true,
+        "output"                     : {
+            "gauss_point"            : [],
+            "nodal_historical"       : ["DISPLACEMENT","ROTATION","VELOCITY","ACCELERATION"],
+            "nodal_non_historical"   : [],
+            "entity"                 : []
+        },
+        "required_variables"         : ["DISPLACEMENT","ROTATION"],
+        "required_dofs"              : ["DISPLACEMENT_X","DISPLACEMENT_Y","DISPLACEMENT_Z","ROTATION_X","ROTATION_Y","ROTATION_Z"],
+        "flags_used"                 : [],
+        "compatible_geometries"      : ["Triangle3D3", "Quadrilateral3D4"],
+        "element_integrates_in_time" : false,
+        "compatible_constitutive_laws": {
+            "type"        : ["PlaneStress"],
+            "dimension"   : ["2D"],
```

Comment:
Again, a membrane is 3D
```suggestion
            "dimension"   : ["3D"],
```

Reply:
But isn't a plane stress a 2D law?

Reply:
A priori yes, but then you have considered a beam as 3D and therefore is incorrect

Reply:
OK changing it (please see #9473)